### PR TITLE
Bundle: export ARCH on Linux

### LINKED ISSUE #3170: Linux bundle build failing
## 🐛 Bug
I know it probably doesn't get used much, but we should either fix the failing Linux bundle build, or skip it for now, since it will make all of our master branch tests look like they're failing.    best I can tell from the [logs](https://github.com/napari/napari/runs/3331141461?check_suite_focus=true),  this might be an issue?
```
-- Running output plugin: appimage -- 
[appimage/stdout] Found appimagetool: /tmp/appimage_extracted_1cdfc4fa2d98741b846c2cf3d6a277cc/plugins/linuxdeploy-plugin-appimage/usr/bin/appimagetool
[appimage/stdout] 
[appimage/stdout] WARNING: appstreamcli command is missing, please install it if you want to use AppStream metadata
[appimage/stderr] Running command: /tmp/appimage_extracted_1cdfc4fa2d98741b846c2cf3d6a277cc/plugins/linuxdeploy-plugin-appimage/usr/bin/appimagetool "/home/runner/work/napari/napari/linux/napari/napari.AppDir" "-g"
[appimage/stderr] 
[appimage/stderr] appimagetool, continuous build (commit 8bbf694), build <local dev build> built on 2020-12-31 11:48:33 UTC
[appimage/stderr] More than one architectures were found of the AppDir source directory "/home/runner/work/napari/napari/linux/napari/napari.AppDir"
[appimage/stderr] A valid architecture with the ARCH environmental variable should be provided
[appimage/stderr] e.g. ARCH=x86_64 appimagetool ...
ERROR: Failed to run plugin: appimage (exit code: 1) 
Error while building app napari.
```

cc @ziyangczi, @goanpeca

### PR DESCRIPTION
**Pull request by jaimergp**
# Description

## Type of change

- [X] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

Closes #3170 

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [X] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

**Review by tlambert03 — APPROVED** [review]
wow, that was it?!

cool.  curious how you figured this out... source code digging?  is there a briefcase note about this?

**Comment by liu-ziyang** [comment]
Nice! I also came up with the same fix earlier but wasn't able to test it on my branch because github won't issue ubuntu 16 runner. Glad it works!

**Comment by liu-ziyang** [comment]
> cool. curious how you figured this out... source code digging? is there a briefcase note about this?

the error message from the failed run did suggest it: 
`[appimage/stderr] More than one architectures were found of the AppDir source directory "/home/runner/work/napari/napari/linux/napari/napari.AppDir"
[appimage/stderr] A valid architecture with the ARCH environmental variable should be provided
[appimage/stderr] e.g. ARCH=x86_64 appimagetool ...`

File: .github/workflows/make_bundle.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ on:
       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
     branches:
       - master
+  pull_request:
+    branches:
+      - master
```
Comment (sofroniewn) [inline .github/workflows/make_bundle.yml:10]: Did this change mean we're now making the bundle on every PR? Is this something we want to keep doing @tlambert03 @jni?

Comment (jaimergp) [reply .github/workflows/make_bundle.yml:10]: I only added this temporarily to have some feedback in the PR itself. I don't think it's needed for every PR and can be reverted safely.

**Comment by jaimergp** [comment]
Btw, in case this is needed again, I created this minimal Dockerfile to debug on my MacOS:

```dockerfile
FROM ubuntu:18.04

RUN apt update && \
    apt install -y wget tar bzip2 git fontconfig libdbus-1-3 libxkbcommon-x11-0 libxcb-icccm4 \
                   libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 \
                   libxcb-xinerama0 libxcb-xfixes0 xvfb file gpg2 appstreamcli

RUN wget -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba && \
    /bin/micromamba create -n default -c conda-forge python=3.8 pip

COPY . /napari

RUN /root/micromamba/envs/default/bin/python -m pip install -e '/napari[all,bundle_build]'

```

File: .github/workflows/make_bundle.yml (hunk 1 above)
Comment (jaimergp) [reply .github/workflows/make_bundle.yml:10]: Hm, maybe it's ok to leave it here? It caught a bug on Windows bundles in [an unrelated PR](https://github.com/napari/napari/pull/3297#issuecomment-909063538).

**Pull request merged**